README.md

```diff
@@ -315,6 +315,7 @@ A curated list of awesome Java frameworks, libraries and software.
 * [JitPack](https://jitpack.io/) - Easy to use package repository for GitHub. Builds Maven/Gradle projects on demand and publishes ready-to-use packages.
 * [Nexus](http://www.sonatype.com/nexus/solution-overview) - Binary management with proxy and caching capabilities.
 * [packr](https://github.com/libgdx/packr/) - Packs JARs, assets and the JVM for native distribution on Windows, Linux and Mac OS X.
+* [really-executable-jars-maven-plugin](https://github.com/brianm/really-executable-jars-maven-plugin) - maven plugin for making chmod +x jar files. Packages a ([standalone executable JAR](https://maven.apache.org/plugins/maven-shade-plugin/examples/executable-jar.html)) as a file that can be run on its own, without having to prefix it with `java -jar` explicitly. Detailed explanation in <http://skife.org/java/unix/2011/06/20/really_executable_jars.html>.
```

Comment:
Too much, you don't have to explain each term. Just write `Maven plugin for making chmod +x jar files.`.